datameta/errors.py

```diff
@@ -53,4 +54,29 @@ def get_validation_error(
         err["message"] = msg
         response_body.append(err)
 
-    return HTTPBadRequest(json=response_body)
+    return base_error(json=response_body)
+
+
+def get_validation_error(
+    messages:List[str],
+    fields:Optional[List[Optional[str]]]=None,
+    entities:Optional[List[Optional[str]]]=None
+) -> HTTPBadRequest:
+    """Generate a Validation Error (400) with custom messages
+    """
+    return get_error(
+        base_error=HTTPBadRequest,
+        messages=messages,
+        fields=fields,
+        entities=entities
+    )
+
+
+def get_not_modifiable_error() -> HTTPForbidden:
+    """Generate a HTTPForbidden (403) error informing 
+    that the resource cannot be modified
+    """
+    return get_error(
+        base_error=HTTPForbidden,
+        messages=["The resource cannot be modified"],
+    )
```

Comment:
Should we maybe make this function more generic? Something like

```suggestion
def get_generic_error(ResponseClass, messages) -> HTTPForbidden:
    """Generate a generic HTTP failure response 
    """
     return get_error(
        base_error=ResponseClass,
        messages=messages,
    )
```

Reply:
But what you are suggesting is identical with the already existing `get_error` class. If you like you can use it like you may use it directly my providing the base error. The `get_not_modifiable_error` is just a shortcut so that you don't have to type the same error message at every location where a "ResoureNotModifiable" Error should be raised.

Reply:
I see. That happens when you look only at diffs ;-)